libc/src/math/cospif16.h

```diff
@@ -0,0 +1,21 @@
+//===-- Implementation header for cospif16 ---------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------===//
```

Comment:
Nit: formatting.

```suggestion
//===-- Implementation header for cospif16 ----------------------*- C++ -*-===//
//
// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===----------------------------------------------------------------------===//
```

You can use https://github.com/overmighty/llvm-dev-utils/blob/master/llvm_libc_add_math_function.py to generate this and other bits of boilerplate. I should probably open a PR at some point to add it to libc/utils and document it.